Annotato/Annotato/SceneDelegate.swift

```diff
@@ -3,7 +3,13 @@ import UIKit
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
-    private let webSocketManager = WebSocketManager()
+    private let webSocketManager = WebSocketManager {
+        guard let userId = AuthViewModel().currentUser?.id else {
+            return ""
+        }
+
+        return RemotePersistenceService.generateWebSocketUrlString(userId: userId)
+    }
```

Comment:
```suggestion
    private let webSocketManager = WebSocketManager(urlStringProducer: {
        guard let userId = AuthViewModel().currentUser?.id else {
            return nil
        }

        return RemotePersistenceService.generateWebSocketUrlString(userId: userId)
    })
```

I was confused at what this closure was at first